fragments/master.sh

```diff
@@ -19,16 +19,15 @@
 
 set -e
 
+source /etc/sysconfig/heat-params
```

Comment:
I think it is possible. I will also remove this dependency from worker.sh because during provisioning we have to save master IP for nodes and use it in worker script.